src/main/client/featureToggleService.ts

```diff
@@ -0,0 +1,25 @@
+import { CoreOptions, RequestAPI } from 'request'
+import { RequestPromise } from 'request-promise-native'
+
+import { FeatureToggleClient } from './featureToggleClient'
+import { FeatureToggleClientFactory } from './featureToggleClientFactory'
+
+export class FeatureToggleService {
+  private featureToggleUri: string
+  private request: RequestAPI<RequestPromise, CoreOptions, CoreOptions>
+
+  constructor (featureToggleApiUri: string,
+               request: RequestAPI<RequestPromise, CoreOptions, CoreOptions>) {
+    this.featureToggleUri = featureToggleApiUri
```

Comment:
Please remove and put `private` modifier to constructor args same way as it has been done here: https://github.com/hmcts/feature-toggle-client/pull/1/files#diff-913161fc5341f17694417539dd5c4723R8